extensions/opentelemetry-exporter-jaeger/runtime/src/main/java/io/quarkus/opentelemetry/exporter/jaeger/JaegerExporterConfig.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.opentelemetry.exporter.jaeger;
+
+import java.time.Duration;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+public class JaegerExporterConfig {
+    @ConfigRoot(name = "opentelemetry.tracer.exporter.jaeger", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+    public static class JaegerExporterBuildConfig {
+        /**
+         * Jaeger SpanExporter support.
+         * <p>
+         * Jaeger SpanExporter support is enabled by default.
+         */
+        @ConfigItem(defaultValue = "true")
+        public Boolean enabled;
+    }
+
+    @ConfigRoot(name = "opentelemetry.tracer.exporter.jaeger", phase = ConfigPhase.RUN_TIME)
+    public static class JaegerExporterRuntimeConfig {
+        /**
+         * The Jaeger endpoint to connect to. The endpoint must start with either http:// or https://.
+         * <p>
+         * Defaults to `http://localhost:14250` if unset.
+         */
+        @ConfigItem(defaultValue = "http://localhost:14250")
+        public String endpoint;
+
+        /**
+         * The amount of time to wait for exporting Spans before an exception is thrown.
+         * <p>
+         * Specify `0` to disable.
+         */
+        @ConfigItem(defaultValue = "10S")
```

Comment:
OK. this needs a better explanation. Are we trying to export spans, and this is thrown when they aren't read? What is throwing and what is it waiting for and what piece is missing? .. This doesn't make sense to me.

Reply:
Maybe I just need to change "for" to "when". Does that help?

It's a timeout on exporting spans to some external thing, in this case the Jaeger collector, which if the timeout is hit an exception is thrown during the span export process

Reply:
When exporting spans to an external collector, throw an exception if this timeout is exceeded. Specify `0` to disable .. (and what, wait forever? fail silently?)

Reply:
The exporter builder has the following definition:
```
  /**
   * Sets the maximum time to wait for the collector to process an exported batch of metrics. If
   * unset, defaults to {@value DEFAULT_TIMEOUT_SECS}s.
   */
```

If timeout is _0_, there is no deadline to complete the gRPC call set, so I believe it would just keep waiting until the connection has an exception

Reply:
How about we draw a little more from their language: 

> The maximum amount of time to wait for the collector to process exported metrics before an exception is thrown. A value of `0` will disable the timeout: the exporter will continue waiting until either exported spans are processed or the connection fails or is closed for some other reason.

or something?

Reply:
Updated